### ISSUE TITLE
syntax highlighting of tasks

### ISSUE THREAD

**Issue by Garonenur** [tomtom/vikitasks_vim#11]
I am not sure, but were tasks highlighted in an older version of viki? Is it possible to get this back if it ever was highlighted?

**Comment by tomtom** [comment]
I'm sorry but I don't quit understand the problem. Aren't tasks highlighted? IIRC the highlighting is done by vikitask, so it has to be loaded. Also tlib should probably be up to date.

BTW as a bit of warning: I moved my todo lists to ttodo_vim (todo.txt format). It could be vikitasks gets unmaintained in the near future.

**Comment by Garonenur** [comment]
my tasks are not highlighted and it seems like indention is the problem there.
I will check out the todo.txt format, does it integrate with VikiDeplate as well?

**Comment by tomtom** [comment]
> I will check out the todo.txt format, does it integrate with 
> VikiDeplate as well?
>

No. But there are many other apps supporting this format, hence the 
locked-in syndrome is attenuated.
